website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+
+<a href="https://www.w3.org/TR/css-align-3/#distribution-values">
+![alignContent references](../static/blog/assets/0.74-align-content.png)
+</a>
+
+## New Architecture support for `position: 'static'`
+
+Elements marked as [`position: 'static'`](https://developer.mozilla.org/en-US/docs/Web/CSS/position#static) may not be offset, and are not considered when determining the [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) of an absolutely positioned element. This allows positioning an element relative to an ancestor which is not its direct parent.
+
+<table>
+
+<tr>
+
+<td width="350">
+
+```jsx
+<View
+  style={{
+    backgroundColor: 'blue',
+    width: 200,
+    height: 200,
+    flexDirection: 'row-reverse',
+  }}>
+  <View
+    style={{
+      backgroundColor: 'red',
+      width: 100,
+      height: 100,
+      position: 'static',
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        width: 25,
+        height: '25%',
+        left: 25,
+        top: 25,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+
+<td width="300">
+
+![Static Example](../static/blog/assets/0.74-static-example.png)
+
+</td>
+
+</tr>
+
+</table>
+Notice how the green `View` declares `left` and `top` and it is positioned relative to the blue `View`, not its parent.
+
+`position: 'static'` may also affect [z-ordering](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) of views. Similar to how `position: 'static'` works with containing blocks, static will not be considered when forming stacking contexts. This allows for stacking an element relative to an ancestor that is not a direct parent.
+
+<table>
+
+<tr>
+
+<td width="350">
+
+```jsx
+<View
+  style={{
+    backgroundColor: 'blue',
+    width: 200,
+    height: 200,
+    flexDirection: 'row',
+  }}>
+  <View
+    style={{
+      backgroundColor: 'orange',
+      width: 50,
+      height: 50,
+      left: 10,
+      position: 'relative',
+      zIndex: 10,
+    }}
+  />
+  <View
+    style={{
+      backgroundColor: 'red',
+      width: 50,
+      height: 50,
+      position: 'static',
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        width: 25,
+        height: 25,
+        zIndex: 11,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+
+<td width="300">
+
+![Static zIndex Example](../static/blog/assets/0.74-zindex-example.png)
+
+</td>
+
+</tr>
+
+</table>
+
+React Native continues to default to `position: 'relative'` when no `position` is set.
+
+## Preparing for the New Architecture
+
+- bridgeless by default?
+- link out to separate New Architecture update?
+- point to New Architecture landing page
```

Comment:
We are preparing working group posts that will be published in `reactwg/react-native-new-architecture`.

I think this section should contain a sample paragraph with the major changes that then refers to the working group. Something like:

```md
In this release, we are making Bridgeless mode the default when the new architecture is enabled. In this way, you can leverage the full power of the New Architecture in your app. You can learn more about the Bridgeless as the Default [in this post](<link_to_the_post>). To make the transition smoother we enhanced the interop layers to cover Bridgeless and we worked with several libraries to make it sure that they work in Bridgeless from day one.

Bridgeless is not the only interop layer we worked on: we improved the New Renderer Interop layer too! Check out the new features [here](<link_to_the_post>).

Finally, if you want to learn more about the New Architecture in general, we moved [the complete documentation](https://github.com/reactwg/react-native-new-architecture/tree/main/docs) from the React Native website to the working group. 
```